frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/cells/select_option_cell/extension.dart

```diff
@@ -112,7 +117,11 @@ class SelectOptionTag extends StatelessWidget {
               name,
               fontSize: FontSizes.s11,
               overflow: TextOverflow.ellipsis,
-              color: AFThemeExtension.of(context).textColor,
+              color: isTextColAlt
+                  ? Theme.of(context)
+                      .extension<AFThemeExtension>()!
+                      .textColorAlt
+                  : Theme.of(context).extension<AFThemeExtension>()!.textColor,
```

Comment:
```suggestion
              color: isTextColAlt
                  ? AFThemeExtension.of(context).textColorAlt
                  : AFThemeExtension.of(context).textColor,
```

Reply:
Is `isTextColAlt` always true? If yes then why do we need it, and if no, then what am I missing?

Reply:
If it's always true, can we calculate the contrast of the default `textColor` and use `textColorAlt` if the contrast is lower than some bound?

Reply:
@Xazin please review my latest changes. I have implemented this logic